src/runtime/tracing.cpp

```diff
@@ -169,7 +170,7 @@ WEAK int32_t default_trace(void *user_context, const halide_trace_event *e) {
 
         {
             ScopedSpinLock lock(&halide_trace_file_lock);
-            halide_print(user_context, ss.str());
+            halide_print(user_context, (const char *)buffer);
```

Comment:
Why is that needed here specifically? Surely there are loads of similar snippets of code in the runtime, that don't have msan annotations, including the printer itself, which is used for everything that gets printed from the runtime (yet I don't remember seeing msan problems from that code).